### DISCUSSION TITLE
HTML / CSS customization

### DISCUSSION THREAD

**Discussion by reneaudet** [knowledgefutures/pubpub#1698] [Support] [answered]
I am working with my first pubs and feel it is lacking some basic customization features (as we may find in WordPress or Drupal). 

- Is there a way to introduce some basic HTML characters? For example, in French, quotation marks require non breakable spaces... (&nbsp;) but I cannot find anyway to insert them.
- I saw that we can customize CSS for a whole community (in an experimental feature), but is it possible to do for a specific collection?
- and finally: is it on your roadmap to offer a multilingual interface to pubpub?

Thanks :)

**Comment by gabestein** [comment, marked as answer]
Hello, welcome, and thanks for your questions. In response to your general comment: PubPub is designed to be WYSIWYM, maintainable without needing any technical expertise, and hard for users to break layouts. The tradeoff is that it is indeed much less customizable than something like Wordpress or Drupal. If fine-grained customization is something you need, you may do better with a self-hosted, fully customizable platform.

In answer to your specific questions:

- At the moment, we don't support non-breaking spaces, although I'll bring it to our developers's attention, as I agree, it would be good to allow for.
- Yes. Collection Layouts have a css class added to their body that you can use to target them. It takes the form of `active-collection-{collection-slug}`. So, a Collection with a slug of `'/my-blog` will have a class of `active-collection-my-blog`.
- Localization is something we'd love to ultimately support, but at the moment, we don't have plans to do so. If we could find a partner and some funding to do it, it's something we would prioritize. At the moment, we focus on internationalization and reducing interface complexity and reliance on strings so that it can be used by non-English speakers as well as possible. Of course, that's not the same thing, but it's what we're able to do for now.

**Reply by reneaudet** [reply]
Hi Gabriel,

Thanks for your quick reply. I understand the philosophy behind PubPub, but a few possibilities for typographical tricks would be a clear advantage, since the visual/graphic rendering is a major aspect in the quality of editing. So thanks for submitting to your developers.

Oh nice, classes associated to collection-slug! It will definitely help.

Localization: I see well what is at stake. But as English is not the only language for science (!), it appears to me a normal suggestion. Nice idea to reduce the imprint of English in the interface.
